work/db/upgrades/5.0.3_cgb/db/02_data/ar-module-data-updates.xml

```diff
@@ -20,14 +20,4 @@
 	xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
 	xsi:schemaLocation='http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd'>
 
-	<changeSet author="KFSCGB" id="KFS_CGB_NEW_AR_DATA">
-		<insert tableName="AR_PMT_MEDIUM_T">
-			<column name="CUST_PMT_MEDIUM_CD" value="LW" />
-			<column name="OBJ_ID" value="KFS-CGB-AR-PMT-MEDIUM-4" />
-			<column name="VER_NBR" value="1" />
-			<column name="CUST_PMT_MEDM_DESC" value="LOC Wire" />
-			<column name="ROW_ACTV_IND" value="Y" />
-		</insert>
-	</changeSet>
-
 </databaseChangeLog>
```

Comment:
If this is the only changeSet in this file, do we want to remove the file completely now (and remove it from the master-data-script.xml)?